### PR TITLE
feat: wormhole total vp 0 fix [web-wormhole]

### PR DESCRIPTION
**Pull request by rachelhox**
## Description

Closes: #XXXX

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] ran linting via `yarn lint`
- [x] wrote tests where necessary
- [x] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [x] targeted the correct branch
- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed
- [x] added a changeset via [`yarn && yarn changeset`](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md)

### CODE REVIEW

File: packages/ui/src/screens/home/components/hero/components/online_voting_power/index.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const OnlineVotingPower: FC<ComponentDefault> = () => {
 
   const votingPowerPercent =
     state.totalVotingPower === 0
-      ? numeral(0)
+      ? numeral((state.activeValidators / state.votingPower) * 100)
```
Comment (teamchong) [inline packages/ui/src/screens/home/components/hero/components/online_voting_power/index.tsx:14]: can we verify for a divide by zero error here?

**Review by teamchong — CHANGES_REQUESTED** [review]

File: packages/ui/src/screens/home/components/hero/components/online_voting_power/index.tsx (hunk 1 above)
Comment (rachelhox) [reply packages/ui/src/screens/home/components/hero/components/online_voting_power/index.tsx:14]: could you clarify? @teamchong

Comment (teamchong) [reply packages/ui/src/screens/home/components/hero/components/online_voting_power/index.tsx:14]: my initial impression is that if the state.votingPower is zero, a division by zero error will be raised.

Comment (rachelhox) [reply packages/ui/src/screens/home/components/hero/components/online_voting_power/index.tsx:20]: so the feature was made for wormhole, and the voting power and active validator set returned from wormhole backend was null 0. But I do see that there might be edge cases in the future, so I have pushed a commit to address your comments, please review @teamchong

**Comment by teamchong** [comment]
the voting power display 0% now

**Review by teamchong — APPROVED** [review]
LGTM

**Pull request merged**